virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -190,73 +197,71 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
+* If you use Portworx as your storage solution on AWS and create a VM disk image, the created image might be smaller than expected due to the filesystem overhead being accounted for twice. (link:https://issues.redhat.com/browse/CNV-32695[*CNV-32695*])
+** As a workaround, you can manually expand the Persistent Volume Claim (PVC) to increase the available space after the initial provisioning process completes.
+
+//4.15 Check with Adam. Issue is closed as "Won't do" but is still a known issue
+* In some instances, multiple virtual machines can mount the same PVC in read-write mode, which might result in data corruption. (link:https://issues.redhat.com/browse/CNV-13500[*CNV-13500*])
 ** As a workaround, avoid using a single PVC in read-write mode with multiple VMs.
 
-//4.14 Leave in per Adam and Stu; will be doc'd in 4.15
-* If you clone more than 100 VMs using the `csi-clone` cloning strategy, then the Ceph CSI might not purge the clones. Manually deleting the clones might also fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055595[*BZ#2055595*])
+//4.15 Have asked Dominik Holler if we still need to keep this
+* If you clone more than 100 VMs using the `csi-clone` cloning strategy, then the Ceph CSI might not purge the clones. Manually deleting the clones might also fail. (link:https://issues.redhat.com/browse/CNV-23501[*CNV-23501*])
 ** As a workaround, you can restart the `ceph-mgr` to purge the VM clones.
 
-//CNV-34198 (BZ 2237287)
-* If you use Portworx as your storage solution on AWS and create a VM disk image, the created image might be smaller than expected due to the filesystem overhead being accounted for twice. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2237287[*BZ#2237287*])
-** As a workaround, you can manually expand the Persistent Volume Claim (PVC) to increase the available space after the initial provisioning process completes.
-
-//BZ2216038 from Ying
-* If you simultaneously clone more than 1000 VMs using the provided DataSources in the `openshift-virtualization-os-images` namespace, it is possible that not all of the VMs will move to a running state. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2216038[*BZ#2216038*])
-** As a workaround, deploy VMs in smaller batches.
 
-//BZ2247593 from Ying
-* Live migration cannot be enabled for a virtual machine instance (VMI) after a hotplug volume has been added and removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055595[*BZ#2247593*])
 
 [discrete]
 [id="virt-4-15-ki-virtualization"]
 ==== Virtualization
-//4.14 Leave in per Stu
-* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037611[*BZ#2037611*])
+//New for 4.15
+* CPU hot unplug for Windows VMs is not supported and results in an error. 
```

Comment:
This may be misleading - we don't yet support any kind of cpu hotplug. @stu-gott

Reply:
@stu-gott Thoughts on the above issue?

Reply:
Concur that this note shouldn't be mentioned in 4.15. CPU Hotplug will GA in 4.16.

Hot unplug not working in Windows is a limitation of that OS. Unsure if that warrants a RN over something in docs as it's a permanent limitation outside of our control.